builder/builder.go

```diff
@@ -522,10 +650,39 @@ func (b *Builder) UpdateMixVer() error {
 	return ioutil.WriteFile(filepath.Join(b.Versiondir, "mixversion"), []byte(strconv.Itoa(mixver+10)), 0644)
 }
 
+func (b *Builder) buildMixBundleDir() error {
+	if err := os.MkdirAll(b.Bundledir, 0777); err != nil {
+		return errors.Errorf("Failed to create dir: %s", b.Bundledir)
+	}
+	// TODO: clear it out if it exists
```

Comment:
What are the consequences of not clearling it out right now? Can we just include that in this PR?

Reply:
Oh whoops! Hahah, completely forgot to go back and implement that. Definitely intended to put that in this PR.